opensearch/src/main/java/org/opensearch/sql/opensearch/request/SubsequentPageRequestBuilder.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.request;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.opensearch.sql.opensearch.data.value.OpenSearchExprValueFactory;
+
+@RequiredArgsConstructor
+public class SubsequentPageRequestBuilder implements PagedRequestBuilder {
```

Comment:
Can we call this `ContinuePageRequestBuilder` to be consistent with `ContinueScrollRequestBuilder`

Reply:
Also: javadoc